library/ssl_cli.c

```diff
@@ -36,7 +36,7 @@
 #endif
 
 #include "mbedtls/ssl.h"
-#include "mbedtls/ssl_internal.h"
+#include "ssl_internal.h"
```

Comment:
No need for some re-ordering? I don't know the include order policy thus this may be fine.

Reply:
We don't have any particular include order policy.